app/marketing/mails.py

```diff
@@ -390,9 +390,12 @@ def bounty_uninterested(to_email, bounty, interest):
 
 def setup_lang(to_email):
     try:
-        from dashboard.models import Profile
-        profile = Profile.objects.filter(email=to_email).first()
-        preferred_language = profile.get_profile_preferred_language()
+        from django.contrib.auth.models import User
+        user = User.objects.select_related('profile').filter(email=to_email).first()
```

Comment:
i didnt realize the `first()` wouldnt throw an exception `User.DoesNotExist`.  instead it just returns `None`!

Reply:
@owocki Yeah.  Whoops.  Forgot to change that to a .get.  I'll switch it up to handle Multiple and DoesNotExist real quick.